DockerOrchestration/ABCIApp/Dockerfile.Dummy

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8
```

Comment:
Do we need a dockerfile at all? when we could run `tendermint node --proxy_app=dummy`